cloud-saas/readme.md

```diff
@@ -1,5 +1,14 @@
 # Glossary of Cloud / SaaS Terms
-A glossary of the terms used in the Cloud and SaaS computing world. Liat Regev and Jonathan Kemp worked on this.
+
+This project is a group project demonstrating creating content in Markdown. It contains a glossary of terms used in the Cloud and Software as a Service (SaaS) discipline. The project adds to the glossary of terms for Our Best Words.
+
+Click on the letter to go to the terms starting with that letter:
+[A](A-D.md#a) &nbsp; [B](A-D.md#a) &nbsp; [C](A-D.md#a) &nbsp; [D](A-D.md#a) &nbsp; [E](A-D.md#a) &nbsp; [F](A-D.md#a) &nbsp; [G](A-D.md#a) &nbsp; [H](A-D.md#a) &nbsp; [I](A-D.md#a) &nbsp; [J](A-D.md#a) &nbsp; [K](A-D.md#a) &nbsp; [L](A-D.md#a) &nbsp;[M](A-D.md#a) &nbsp; [N](A-D.md#a) &nbsp; [O](A-D.md#a) &nbsp; [P](A-D.md#a) &nbsp; [Q](A-D.md#a) &nbsp; [R](A-D.md#a) &nbsp; [S](A-D.md#a) &nbsp; [T](A-D.md#a) &nbsp; [U](A-D.md#a) &nbsp; [V](A-D.md#a) &nbsp; [W](A-D.md#a) &nbsp; [X](A-D.md#a) &nbsp; [Y](A-D.md#a) &nbsp; [Z](A-D.md#a) &nbsp;
```

Comment:
This is not going to work for a few reasons
1. you're pointing to a file which you don't have (A-D.md doesn't exist in your directory and it shouldn't because your content is all inside the readme.
2. Instead you need to link to the sections below. For example [A](#a) &nbsp; [B](#b) &nbsp; and so on
3. IF there is no heading for the letter simply write the letter as follows: C &nbsp;